apps/communications/contacts/style/contacts.css

```diff
@@ -426,7 +425,7 @@ gaia-header [data-icon] {
 #import-settings .view-body-inner,
 #view-contact-form .view-body-inner,
 #ice-settings .view-body-inner {
-  background: #f4f4f4;
+  background: #ffffff;
```

Comment:
For some cases we received visual as well, thats why we did this changed.
Complete Visual design specs implementation going on right now.